Doc/library/typing.rst

```diff
@@ -779,32 +779,25 @@ The module defines the following classes, functions and decorators:
 
    .. versionadded:: 3.5.2
 
-.. class:: io
+.. class:: IO
+           TextIO
+           BinaryIO
 
-   Wrapper namespace for I/O stream types.
-
-   This defines the generic type ``IO[AnyStr]`` and subclasses ``TextIO``
-   and ``BinaryIO``, deriving from ``IO[str]`` and ``IO[bytes]``,
-   respectively. These represent the types of I/O streams such as returned by
+   Generic type ``IO[AnyStr]`` and subclasses ``TextIO(IO[str])``
+   and ``BinaryIO(IO[bytes])``
```

Comment:
I actually like the original formulation more. But this one is also fine.

Reply:
Maybe "and its subclasses".

Reply:
The problem is that "This defined" is now wrong, but when removing the verb you have to reformulate. I first ended up with:

> The generic type `IO[AnyStr]` and subclasses `TextIO` and `BinaryIO`, deriving from `IO[str]` and `IO[bytes]`, respectively, represent the types of I/O streams such as returned by `open`.

 But I found this hard to understand.

Reply:
@srittau You can also try splitting the sentence in two parts. Anyway, this is not very important.